src/shogun/kernel/GaussianKernel.h

```diff
@@ -120,7 +120,29 @@ class CGaussianKernel: public CShiftInvariantKernel
 	 */
 	SG_FORCED_INLINE float64_t get_width() const
 	{
-		return std::exp(m_log_width * 2.0) * 2.0;
+		return CGaussianKernel::from_log_width(m_log_width);
+	}
+
+	/**
+	 * Converts width to log domain.
```

Comment:
this is not quite precise. I would actually ditch the log in the conversion and rather convert from `$-(x-y)^2/\gamma$` parametrisation to `$(-(x-y)^2 / {2\sigma^2})$` (and back), i.e. there must be a square root and a division by two. Then add a log around this explicitly. The log parametrisation has different reasons, and also putting/removing a log is trivial and doesnt twist your brain.

Reply:
Oh and whatever it does, make sure to add the math in the doxygen so it is clear

Reply:
I am not sure I remember what this comment was about. Shouldn't we have this helper to convert width <-> log_width?

Reply:
yes, and also maybe converting between th edifferent parametrisations (see above)
Just to keep the code readable ....

Reply:
I see, I was completely missing the point, I was just seeing the log :D So this should be a function to convert width <-> gamma and the log is handled internally?

Reply:
should we register `length_scale`/`width` instead of `log_width` then? And then just have a callback around `width` that updates `log_width` when width is changed? And `log_width` is hidden from the user?

Reply:
I'd prefer registering the bandwidth itself (the actual quantity from the formula) 
Beware that this requires some refactoring work ....

Reply:
I'll can refactor that in a different PR, or start an issue